src/Services/Logger.ts

```diff
@@ -31,7 +31,7 @@ const esTransportOpts = {
     } as ClientOptions,
 };
 
-if (config.env == "prod") {
+if (config.env == "production" && config.logging.es.enabled) {
```

Comment:
This ain't gonna work.
Better this
```suggestion
if (config.env == "production" && config.logging.es === "enabled") {
```